### PR TITLE
[iDEAL] iDEAL Button

### PR DESCRIPTION
**Pull request by mpetrenco**
- Added the iDEAL button as part of the Payment Methods screen;

### CODE REVIEW

**Comment by mpetrenco** [comment]
'./clang-format.sh' has been done in a latter PR to avoid adding additional changes not related to the iDEAL button.

File: Source/Controller/JudoPaymentMethodsViewController.m [hunk 1]
```diff
@@ -69,101 +70,137 @@ - (instancetype)initWithTheme:(JPTheme *)theme
...
+        
+        NSBundle *bundle = [NSBundle bundleForClass:JudoKit.class];
+        
+        NSString *iconBundlePath = [bundle pathForResource:@"icons" ofType:@"bundle"];
+        NSBundle *iconBundle = [NSBundle bundleWithPath:iconBundlePath];
+        
+        NSString *iconFilePath = [iconBundle pathForResource:@"logo-ideal" ofType:@"png"];
+        
+        [idealButton setImage:[UIImage imageWithContentsOfFile:iconFilePath]
+                     forState:UIControlStateNormal];
```
Comment (stpopa) [inline Source/Controller/JudoPaymentMethodsViewController.m]: you use this logic in 2 places, let's extract the retrieval of the image form the `iconBundlePath` in a separate extension, I believe we already have the `NSBundle+Additions`, so let's add it there

File: Source/Controller/JudoPaymentMethodsViewController.m [hunk 2]
```diff
@@ -69,101 +70,137 @@ - (instancetype)initWithTheme:(JPTheme *)theme
...
+        
+        idealButton.imageView.contentMode = UIViewContentModeScaleAspectFit;
+        
+        [idealButton setTitle:@"iDEAL" forState:UIControlStateNormal];
+        [idealButton setTitleColor:UIColor.whiteColor forState:UIControlStateNormal];
+        idealButton.titleLabel.font = [UIFont systemFontOfSize:20.0 weight:UIFontWeightBold];
```
Comment (stpopa) [inline Source/Controller/JudoPaymentMethodsViewController.m]: let's get the font from the theme `[idealButton.titleLabel setFont:self.theme.buttonFont];`

File: Source/Controller/JudoPaymentMethodsViewController.m (hunk 1 above)
Comment (mpetrenco) [reply Source/Controller/JudoPaymentMethodsViewController.m]: Done.

File: Source/Controller/JudoPaymentMethodsViewController.m (hunk 2 above)
Comment (mpetrenco) [reply Source/Controller/JudoPaymentMethodsViewController.m]: Done.

**Pull request merged**